pragati/OnlineShopping/contact.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>contact us</title>
+    <link
```

Comment:
tags should be on one line why is it like this

<link
      rel="stylesheet"
      href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"
    />
it should be 
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"/>